layers/org/README.org

```diff
@@ -59,6 +60,21 @@ at this stage.  It is possible to disable the configuration of gnuplot support
 as usual by adding the package =gnuplot= to your =dotspacemacs-excluded-packages=
 variable.
 
+** Reveal.js support
+
+To enable the export of your org file as [[http://lab.hakim.se/reveal-js/][reveal.js]] presentation, set the
+variable =org-enable-reveal-js= to =t=. This will install [[https://github.com/yjwen/org-reveal/][org-reveal]] extension.
+
```

Comment:
"To enable the export of your org file as a [[http://lab.hakim.se/reveal-js/][reveal.js]] presentation, set the variable =org-enable-reveal-js= to =t=. This will install the [[https://github.com/yjwen/org-reveal/][org-reveal]] extension."